delta/sdk-testkit/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/testkit/ProjectsDummy.scala

```diff
@@ -0,0 +1,96 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.testkit
+
+import java.util.UUID
+
+import akka.persistence.query.Offset
+import cats.effect.Clock
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectCommand.{CreateProject, DeprecateProject, UpdateProject}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRejection.UnexpectedInitialState
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectState.Initial
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects._
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.{Pagination, SearchParams, SearchResults}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{BaseUri, Envelope}
+import ch.epfl.bluebrain.nexus.delta.sdk.testkit.ProjectsDummy.{ProjectsCache, ProjectsJournal}
+import ch.epfl.bluebrain.nexus.delta.sdk.utils.UUIDF
+import ch.epfl.bluebrain.nexus.delta.sdk.{Organizations, ProjectResource, Projects}
+import monix.bio.{IO, Task, UIO}
+
+/**
+ * A dummy Projects implementation
+ * @param journal       the journal to store events
+ * @param cache         the cache to store resources
+ * @param organizations an Organizations instance
+ */
+final class ProjectsDummy private (journal: ProjectsJournal,
+                                   cache: ProjectsCache,
+                                   organizations: Organizations)
+                                  (implicit base: BaseUri,
+                                   clock: Clock[UIO],
+                                   uuidf: UUIDF)
+  extends Projects {
+
+  override def create(ref: ProjectRef,
+                      fields: ProjectFields)(implicit caller: Identity.Subject): IO[ProjectRejection, ProjectResource] =
+    eval(CreateProject(ref, fields.description, fields.apiMappings, fields.baseOrGenerated(ref), fields.vocabOrGenerated(ref), caller
+    )).flatMap(cache.setToCache)
```

Comment:
I see you set the elements to the cache on each method (create/update/deprecate) instead of doing it inside the `eval` for comprehension.

This is fine, but If you do it this way, can you also change it in `OrganizationsDummy`?